project-base/src/Controller/Front/CartController.php

```diff
@@ -138,13 +140,18 @@ public function indexAction(Request $request)
         ]);
     }
 
-    public function boxAction()
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     */
+    public function boxAction(Request $request)
     {
         $orderPreview = $this->orderPreviewFactory->createForCurrentUser();
 
         return $this->render('Front/Inline/Cart/cartBox.html.twig', [
             'cart' => $this->cartFacade->findCartOfCurrentCustomerUser(),
             'productsPrice' => $orderPreview->getProductsPrice(),
+            'isIntentActive' => $request->get('isIntentActive'),
+            'pagesWithDisabledCartHover' => self::PAGES_WITH_DISABLED_CART_HOVER,
```

Comment:
You are right, I would like do it, but I had  problem with get route name in this controller, because boxAction in this controller is called in `header.twig.html` as `render(controller....CartController:boxAction)` and I was not able get real route name in controller (I tried get route name from Request). 

Do you know how can I get real route neme?

Reply:
I see... 
It's possible to inject `RequestStack` and use masterRequest as `$this->requestStack->getMasterRequest()->get('_route'));`

the same way as it's used in `\Shopsys\FrameworkBundle\Twig\RequestExtension::getRoute

Reply:
Thak you!